File: numeric-keyboard/src/main/java/com/konaire/numerickeyboard/util/CustomEditText.java
package com.konaire.numerickeyboard.util;

import android.content.Context;
import android.graphics.Typeface;
import android.util.AttributeSet;

import androidx.annotation.Nullable;
import androidx.appcompat.widget.AppCompatEditText;

public class CustomEditText extends AppCompatEditText {
    private Context ctx;
    public CustomEditText(Context context) {
        super(context);
        ctx=context;
        init();
    }

    public CustomEditText(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
        ctx=context;
        init();
    }

    public CustomEditText(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        ctx=context;
        init();
    }

    public void init()
    {
            Typeface face=Typeface.createFromAsset(ctx.getAssets(), "fonts/avenir_medium.ttf");
            setTypeface(face);
    }

}

File: app/src/main/java/com/rap/sheet/utilitys/CustomEditText.kt
package com.rap.sheet.utilitys

import android.content.Context
import android.graphics.Typeface
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatEditText
import androidx.core.content.res.ResourcesCompat
import com.rap.sheet.R
import com.rap.sheet.extenstion.regularFont

class CustomEditText : AppCompatEditText {


    constructor(context: Context) : super(context) {
        init()

    }

    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {

        init()
    }

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {
        init()
    }

    fun init() {
        if (!isInEditMode) {
            try {
                typeface = context.regularFont()
            } catch (e: Exception) {
                e.printStackTrace()
            }
        }
    }


}